uint8 type
uint32 data
uint8 ROBOT_CHECK=0
uint8 FREE_ROBOT=1
uint8 CHARGE_BATTERY=2
uint8 CANCEL_TASK=3
uint8 PERFORMING_PICK_UP=4
uint8 PERFORMING_DELIVERY=5
uint8 SUCESS_TASK=6
uint8 FAIL_TASK=7